CHANGELOG.md

```diff
@@ -48,6 +48,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 [#5553]: https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5553
 [#5554]: https://github.com/open-telemetry/opentelemetry-go-contrib/issues/5554
 
+### Changed
+
+- Use more efficient `WithAttributeSet()` in `otelhttp` and `otelgrpc`. (#5664) 
```

Comment:
I would phrase this as being purely a performance optimization, not a feature-visible change.

```suggestion
- Improve performance of `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc` and `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp` with the usage of `WithAttributeSet()` instead of `WithAttribute()`. (#5664) 
```